TypeCobol/Compiler/SqlAnalysis/ASTRules/CommitStatementNotAllowed.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Analysis;
+using TypeCobol.Compiler.Parser;
+using TypeCobol.Compiler.Diagnostics;
+using TypeCobol.Compiler.SqlNodes;
+
+namespace TypeCobol.Compiler.SqlAnalysis.ASTRules
+{
+    /// <summary>
+    /// Sql Commit Statement not allowed rule class
+    /// </summary>
+    internal class CommitStatementNotAllowed : SqlAbstractRuleChecker<Commit>
+    {
+        public CommitStatementNotAllowed(Action<Violation> addViolation) : base("cobol:SqlCommitRollback", addViolation)
```

Comment:
This is for the POC only. This goes into our private analyzer. While the parser defines the notion of `Violation`, it does not have any knowledge about rules.

Reply:
Ok it will be a task

Reply:
removed in [7f2afb7](https://github.com/TypeCobolTeam/TypeCobol/pull/1928/commits/7f2afb778747d8f26ff0ce751781c2ab8845aa78)